src/test/java/bio/terra/service/dataset/DatasetAzureIntegrationTest.java

```diff
@@ -454,19 +457,42 @@ public void datasetIngestFileHappyPath() throws Exception {
     TestUtils.verifyHttpAccess(snapshotParquetUrl, Map.of());
     verifySignedUrl(snapshotParquetUrl, steward(), "rl");
 
+    List<String> drsIds = new ArrayList<>();
     for (AccessInfoParquetModelTable table : snapshotParquetAccessInfo.getTables()) {
       if (tablesToCheck.contains(table.getName())) {
         String tableUrl = table.getUrl() + "?" + table.getSasToken();
         TestUtils.verifyHttpAccess(tableUrl, Map.of());
         verifySignedUrl(tableUrl, steward(), "rl");
+
+        // The vocabulary table has file data so test Drs on that one
+        // TODO: once we have an endpoint to expose parquet data, we should use that mechanism here
+        if (table.getName().equals("vocabulary")) {
+          List<Map<String, String>> records = ParquetUtils.readParquetRecords(tableUrl);
+          assertThat("2 rows are present", records.size(), equalTo(2));
+          // Extract the DRS Ids
+          drsIds.addAll(
+              records.stream()
+                  .map(r -> r.get("vocabulary_reference"))
+                  .collect(Collectors.toList()));
+        }
       }
     }
 
+    // Assert that 2 drs ids were loaded
+    assertThat("2 drs ids are present", drsIds.size(), equalTo(2));
+    // Ensure that all DRS can be parsed
+    List<String> drsObjectIds =
+        drsIds.stream()
+            .map(DrsIdService::fromUri)
+            .map(DrsId::toDrsObjectId)
+            .collect(Collectors.toList());
+
     String fileId = result.getLoadFileResults().get(0).getFileId();
     String filePath = result.getLoadFileResults().get(0).getTargetPath();
 
     // Do a Drs lookup
     String drsId = String.format("v1_%s_%s", snapshotId, fileId);
```

Comment:
Whenever I see code building IDs from scratch like this I wonder what we'll do if we ever change the format. The format itself is supposed to be opaque so users shouldn't be depending on it. Our tests should be written the same way and use a (non-test) Util method to build DRS IDs instead of doing it by hand like this does.

Reply:
I disagree here too.  Then the test isn't testing anything.  This has the benefit of extra-reminding the developer that that if the format changes, then it's going to break things downstream

Reply:
Since there's no way to refactor the SQL code anyway there's not much we can do to eliminate the duplication here. But I disagree with the philosophy that backward compatibility should be tested as a side effect of a test of something else. I'd prefer to have a test that's explicitly for compatibility instead, if we think that's an important thing to do.